### PR TITLE
Fix syntax for gatsby-remark-responsive-iframe plugin.

### PR DESCRIPTION
**Pull request by levino**
This helps getting over the current error message. Looks like the syntax for the plugin has changed from `1.x` to `2.x`. https://www.npmjs.com/package/gatsby-remark-responsive-iframe

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
-    "plugins": ["lodash"]
-}
```
Comment (levino) [inline .babelrc:3]: Having a `.babelrc` [totally crashes gatsby v2](https://www.gatsbyjs.org/docs/migrating-from-v1-to-v2/#move-babel-configuration). It is "forbidden" to have a `.babelrc` at the root:

File: gatsby-config.js [hunk 2]
```diff
@@ -44,7 +44,10 @@ module.exports = {
             }
           },
           {
-            resolve: 'gatsby-remark-responsive-iframe'
+            resolve: 'gatsby-remark-responsive-iframe',
+            options: {
+              plugins: [`gatsby-remark-responsive-iframe`]
+            }
```
Comment (levino) [inline gatsby-config.js:50]: Looks like the plugin [has a new syntax](https://www.npmjs.com/package/gatsby-remark-responsive-iframe#how-to-use).

File: gatsby-node.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 const path = require('path')
 const _ = require('lodash')
-const webpackLodashPlugin = require('lodash-webpack-plugin')
+const LodashModuleReplacementPlugin = require('lodash-webpack-plugin')
```
Comment (levino) [inline gatsby-node.js:3]: This is now a [constructor](https://github.com/lodash/lodash-webpack-plugin#usage)

File: gatsby-node.js [hunk 4]
```diff
@@ -111,8 +111,8 @@ exports.createPages = ({ graphql, actions }) => {
   })
 }
 
-exports.modifyWebpackConfig = ({ config, stage }) => {
+exports.onCreateWebpackConfig = ({ actions, stage }) => {
```
Comment (levino) [inline gatsby-node.js:114]: New [API](https://www.gatsbyjs.org/docs/migrating-from-v1-to-v2/#change-modifywebpackconfig-to-oncreatewebpackconfig). Part of the migration guide.

File: src/components/SocialLinks/SocialLinks.jsx [hunk 5]
```diff
@@ -1,5 +1,22 @@
...
+  TwitterIcon,
+  TelegramIcon,
+  GooglePlusIcon,
+  LinkedinIcon,
+  RedditIcon,
+  FacebookShareCount,
+  GooglePlusShareCount,
+  LinkedinShareCount,
+  RedditShareCount
+} from 'react-share'
```
Comment (levino) [inline src/components/SocialLinks/SocialLinks.jsx:19]: `react-share` has a [new API](https://github.com/nygardk/react-share/blob/HEAD/migrate-v1-to-v2.md) in v2.

File: src/components/UserLinks.jsx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
-import { FaGithubAlt, FaTwitter } from 'react-icons/lib/fa'
-import { MdMessage, MdEmail } from 'react-icons/lib/md'
+import { FaGithubAlt, FaTwitter } from 'react-icons/fa'
+import { MdMessage, MdEmail } from 'react-icons/md'
```
Comment (levino) [inline src/components/UserLinks.jsx:3]: `react-icons` have [new API](https://www.npmjs.com/package/react-icons#migrate-from-version-2---3)

**Comment by glauberfc** [comment]
Very good work, man! Thanks!

**Pull request merged**